### PR TITLE
Feature/Signup-UI

### PR DESCRIPTION
**Pull request by xuanquoche**
image

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+  max-width: 1440px;
+  /* margin: 0 auto; */
```
Comment (baonguyen26) [inline client/src/App.css]: remove

File: client/src/components/common/Input/style.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.iput {
```
Comment (baonguyen26) [inline client/src/components/common/Input/style.css]: ?

File: client/src/components/common/Input/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, { memo } from "react";
+import mailLogo from "../../../assets/icon/mail.png";
+import './style.css'
+
+interface InputPropsCus extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size'> {
+  label?: string;
+  className?: string;
+  size?: "large" | "small" | "medium" | string;
```
Comment (haunguyen15-tt) [inline client/src/components/common/Input/index.tsx:11]: I think we should define an enum here

File: client/src/components/common/Input/index.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  icon?: string;
+  placeholder?: string;
+  type: string
+}
+
+
+export const Input: React.FC<InputPropsCus> = memo(
+  ({ label, className, size, placeholder,type, ...rest }) => {
```
Comment (haunguyen15-tt) [inline client/src/components/common/Input/index.tsx]: Check code format

File: client/src/components/common/Input/style.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    border: none;
+    outline: none;
+    width: 100%;
+    background-color: transparent;
+}
+
+.input-group{
+    background-color:#F4F4F4 ; padding: 10px;
```
Comment (haunguyen15-tt) [inline client/src/components/common/Input/style.css]: Define variable

File: client/src/components/common/TickList/index.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import './style.css';
+interface TickProps {
+    icon?: string,
+    text: string,
+    clasname?: string
```
Comment (haunguyen15-tt) [inline client/src/components/common/TickList/index.tsx:10]: className

File: client/src/components/common/TickList/style.css [hunk 7]
```diff
@@ -0,0 +1,3 @@
+.text-tick{
+    color: #6F767E;
+}
```
Comment (haunguyen15-tt) [inline client/src/components/common/TickList/style.css]: End of line

File: client/src/pages/Login/login.css [hunk 8]
```diff
@@ -0,0 +1,35 @@
+.wrapp{
+    min-width: 370px;
+    height: 100vh;
+}
+
+.content{
+    max-width: 370px;
+    margin: auto auto;
```
Comment (haunguyen15-tt) [inline client/src/pages/Login/login.css]: margin: auto

File: client/src/pages/Login/login.css [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+@media screen and (min-width: 1240px) {
+    .sidebar{
+        width: 30%;
+    }
+    .content{
+        width: 70%;
+    }
+}
```
Comment (haunguyen15-tt) [inline client/src/pages/Login/login.css:36]: I think we should use variable for width. And when the breakpoint change, just change the variable

**Review by haunguyen15-tt — CHANGES_REQUESTED** [review]
Check the code format. And define variable for color, font size, ...

File: client/src/components/common/Input/index.tsx (hunk 4 above)
Comment (xuanquoche) [reply client/src/components/common/Input/index.tsx]: what did you mean? can you tell me more detail?

Comment (haunguyen15-tt) [reply client/src/components/common/Input/index.tsx]: Just saw placeholder and type are have not spacing

**Pull request closed without merging**